src/main/resources/l10n/JabRef_fr.properties

```diff
@@ -1175,7 +1175,7 @@ Show_icons_for_groups=Afficher_les_icônes_pour_les_groupes
 Show_last_names_only=Afficher_uniquement_les_noms_propres
 
 Show_names_unchanged=Ordre_des_noms_inchangé
-
+        
```

Comment:
Good catch, I can strip (deleting spaces before and after) each line in the English file (thus in each file).